src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/LogicalOperator.java

```diff
@@ -21,4 +21,6 @@ public interface LogicalOperator
 	 * Returns an identifier of this operator, which should be unique for all the operators within the same plan.
 	 */
 	int getID();
+	
+	String printString(String indentStr);
```

Comment:
This should not be part of the operators themselves. Instead, the decision of how exactly each operator is presented should be part of the plan printer. So, please remove this function here (and for the purpose of this particular PR, the plan printer can still just use the `toString()` function of the operators; a later PR can then change this behavior by doing something nicer looking directly in the plan printer).